### PR TITLE
115 add flye to nf corebacass

### PR DESCRIPTION
**Pull request by lborcard**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bacass/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/bacass _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X] Usage Documentation in `docs/usage.md` is updated.
- [X] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -164,7 +165,7 @@
                 },
                 "dfast_config": {
                     "type": "string",
-                    "default": "assets/test_config_dfast.py",
+                    "default": "/storage/tmp/scratch_lb21z855/nf-core-hackathon24/bacass/assets/test_config_dfast.py",
```
Comment (maxulysse) [inline nextflow_schema.json:168]:
```suggestion
                    "default": "assets/test_config_dfast.py",
```

**Comment by Daniel-VM** [comment]
The " does not match the template" errors are on my side, I believe.  I still have to make the PR  with the nf-core 2.13.1 template update.

**Comment by lborcard** [comment]
> Can you update CHANGELOG as well?

ok I added this pull request to the changelog

**Review by lborcard — COMMENTED** [review]
nothing to report

**Comment by lborcard** [comment]
I dont understand what happened to the template?

**Comment by Daniel-VM** [comment]
I need to first make a PR and merge into dev the nf-core tools template v2.13.1 (hopefully in a couple of hours). Since  current version of nf-core/bacass  is bit older, the git CI, which relies on latest nf-core tools is rasing errors due to missing or unexpected files/file-contents.

**Comment by lborcard** [comment]
> I need to first make a PR and merge into dev the nf-core tools template v2.13.1 (hopefully in a couple of hours). Since current version of nf-core/bacass is bit older, the git CI, which relies on latest nf-core tools is rasing errors due to missing or unexpected files/file-contents.

Ok so what should I do in the meantime?

**Comment by Daniel-VM** [comment]
I was reviewing your changes and I found that most dev implementations such as dragonflye disappeared 🤔.  Wait,  and also changelog addition should be added to section : v2.2.0dev nf-core/bacass: "Aqua Platinum Zebrafish".  Bit it doesn't exist.... 

This in comming branch is not from your forked but from another one directly created in the original nf-core/bacass repo (@d4straub  ⚠️ ).

When attepting to make PR you sould do it form your` **forked_repo:branch_name**` to `**nf-core_original_repo:dev**`. See this one: #117 .

**Comment by lborcard** [comment]
Sorry about that it kind of Naturally created this new branch when I used the nf-core tools and I didn't see this section of the instructions. My mistake I will fork it and merge my changes and start the process again. My apologies.

**Comment by Daniel-VM** [comment]
No worries. I am closing this PR now. Remember: when working locally, checkout first to your 'dev' branch, then create a new branch. This way, your new branch will have the latest development code, reducing the probability of conflicts when making a PR against nf-core:dev.

**Pull request closed without merging**